packages/types/lib/IDataSource.ts

```diff
@@ -3,6 +3,7 @@ import type { IActionContext } from './IActionContext';
 
 export type IDataSource = string | RDF.Source | {
   type?: string;
+  ignoreSparqlEndpointFromServiceDescription?: boolean;
```

Comment:
Instead of storing this in a new field, I would suggest storing this in the context, by introducing a new context key in the `context-entries` package. This would also align nicely with the new CLI argument by handling it in `CliArgsHandlerQuery`.